tools/depends/target/pythonmodule-pil/Makefile

```diff
@@ -64,9 +69,10 @@ endif
 ifeq ($(OS),android)
 	cd $(PILPATHLIB) && unzip -o ../Pillow-*.egg
 	cd $(PILPATHLIB)/PIL && \
-          sed -i -e 's/import sys/import os, sys /' \
-                 -e '/__file__/ s/_imaging/lib_imaging/g' \
-                 -e 's/pkg_resources.resource_filename(__name__,/os.path.join(os.environ["KODI_ANDROID_LIBS"], /'  _imaging*.py
+          sed $(SED_FLAG) -e 's/import sys/import os, sys /' \
```

Comment:
using plain variable should be enough: `$SED_FLAG`

Reply:
I'll be honest, I prefer to have braces. Delimits var names in case of concatenation, and just easier to look for imo

Reply:
I prefer braces too

Reply:
only now it occurred to me that this is a makefile, not a plain shell script... Sorry for the noise!